### PR TITLE
Add test for SponsorStore

### LINKED ISSUE #723: Add Sponsor Store test
## Kind (Required)

- Other

 ## Overview (Required)

- You can see other screen tests for your reference.

 ## Links	

- SponsorStore https://github.com/DroidKaigi/conference-app-2019/blob/master/feature/sponsor/src/main/java/io/github/droidkaigi/confsched2019/sponsor/ui/store/SponsorStore.kt#L13
- AnnouncementStoreTest https://github.com/DroidKaigi/conference-app-2019/blob/master/feature/announcement/src/test/java/io/github/droidkaigi/confsched2019/announcement/ui/store/AnnouncementStoreTest.kt#L20

### PR DESCRIPTION
**Pull request by hiji**
## Issue
- close #723

## Overview (Required)
- Add test for `SponsorStore`
- Refer to `AnnouncementStoreTest`

## Links
- 

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: feature/sponsor/src/test/java/io/github/droidkaigi/confsched2019/SponsorDummyDatas.kt [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+import com.soywiz.klock.DateTime
+import com.soywiz.klock.hours
+import io.github.droidkaigi.confsched2019.model.SponsorCategory
+import io.github.droidkaigi.confsched2019.model.Sponsor
+
+private val startTime =
+    DateTime.createAdjusted(2019, 2, 7, 10, 0).toOffsetUnadjusted(9.hours).utc
+
+fun dummySponsorsData(): List<SponsorCategory> {
```
Comment (takahirom) [inline feature/sponsor/src/test/java/io/github/droidkaigi/confsched2019/SponsorDummyDatas.kt]: 👍

File: feature/sponsor/src/test/java/io/github/droidkaigi/confsched2019/sponsor/ui/store/SponsorStoreTest.kt [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        val observer = mockk<(LoadingState?) -> Unit>(relaxed = true)
+
+        sponsorStore.loadingState.changedForever(observer)
+        verify { observer(LoadingState.LOADING) }
+
+        dispatcher.dispatch(Action.SponsorLoadingStateChanged(LoadingState.LOADED))
+        verify { observer(LoadingState.LOADED) }
+    }
+
+    @Test fun announcements() = runBlocking {
```
Comment (takahirom) [inline feature/sponsor/src/test/java/io/github/droidkaigi/confsched2019/sponsor/ui/store/SponsorStoreTest.kt]: sponsors 📛

Comment (hiji) [reply feature/sponsor/src/test/java/io/github/droidkaigi/confsched2019/sponsor/ui/store/SponsorStoreTest.kt]: @takahirom I am sorry ... I used the test class that I referred to as it was. I fixed it.

**Comment by takahirom** [comment]
Thank you for your contribution!

**Pull request merged**